src/System.Text.Json/tests/JsonElementValueTests.cs

```diff
@@ -0,0 +1,90 @@
+
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Tests
+{
+    public static class JsonElementValueTests
```

Comment:
Why a new file?

Reply:
The new method should be added to all the existing tests which throw when the wrong type is present or the value is default or the document is disposed.

Reply:
> Why a new file?

I wanted to add tests for JsonElement.ValueTests into an existing JsonElementTests file, but realized there is currently files like JsonElementCloneTests and JsonElementWriteTests files present so I instead went with being consistent and adding another file.

I can move elsewhere if you have other recommendations to where to have them

Reply:
JsonDocumentTests is where all of the other value-based tests are, this seems like a good fit there.

Reply:
> JsonDocumentTests is where all of the other value-based tests are, this seems like a good fit there.

OK

> The new method should be added to all the existing tests which throw when the wrong type is present or the value is default or the document is disposed.

Could you please clarify this comment? Which method should be added?

Reply:
`ValueEquals`.  All the places that call GetString() and expect it to throw should also call ValueEquals and expect it to throw.

Reply:
Oh I see. Thank you for clarification.